packages/api/lib/writeRecords/write-granules.js

```diff
@@ -207,66 +137,33 @@ const getGranuleCumulusIdFromQueryResultOrLookup = async ({
 };
 
 /**
- * Write a granule to DynamoDB and PostgreSQL
+ * Write a granule to PostgreSQL
  *
  * @param {Object} params
- * @param {Object} params.granule - An API granule object
- * @param {Object} params.processingTimeInfo
- *   Processing time information for the granule, if any
- * @param {Object} params.error - Workflow error, if any
- * @param {string} params.workflowStartTime - Workflow start time
- * @param {string} params.workflowStatus - Workflow status
- * @param {Object} params.queryFields - Arbitrary query fields for the granule
+ * @param {Object} params.granuleRecord - An postgres granule records
  * @param {string} params.collectionCumulusId
  *   Cumulus ID for collection referenced in workflow message, if any
- * @param {string} params.providerCumulusId
- *   Cumulus ID for provider referenced in workflow message, if any
  * @param {string} params.executionCumulusId
  *   Cumulus ID for execution referenced in workflow message, if any
- * @param {string} params.pdrCumulusId
- *   Cumulus ID for PDR referenced in workflow message, if any
  * @param {Knex.transaction} params.trx - Transaction to interact with PostgreSQL database
- * @param {string} params.updatedAt - Update timestamp
- * @param {Array} params.files - List of files to add to Dynamo Granule
  *
  * @returns {Promise<number>} - Cumulus ID from PostgreSQL
  * @throws
  */
-const _writeGranuleViaTransaction = async ({
-  granule,
-  processingTimeInfo,
-  error,
-  workflowStartTime,
-  workflowStatus,
-  queryFields,
+const _writePostgresGranuleViaTransaction = async ({
+  granuleRecord,
   collectionCumulusId,
-  providerCumulusId,
   executionCumulusId,
-  pdrCumulusId,
   trx,
-  updatedAt,
-  files,
+  granulePgModel = new GranulePgModel(),
```

Comment:
```suggestion
  granulePgModel,
```

Initialization is unnecessary since it already should be initialized by calling code